.resinci.json

```diff
@@ -0,0 +1,47 @@
+{
+  "dependencies": {
+    "linux": [
+      "libudev-dev",
+      "libusb-1.0-0-dev",
+      "libyaml-dev"
+    ]
+  },
+  "builder": {
+    "appId": "io.resin.etcher",
+    "copyright": "Copyright 2016 Resinio Ltd",
+    "productName": "Etcher",
+    "mac": {
+      "category": "public.app-category.developer-tools"
+    },
+    "dmg": {
```

Comment:
Should "dmg" be listed _inside_ "mac"?

Reply:
This is passed as-is to electron-builder, which expects mac to be outside dmg (not up to us). We will put a layer on top of electron-builder at some point, though, so we will address this soon